packages/terra-date-picker/translations/ar.json

```diff
@@ -1,5 +1,6 @@
 {
   "Terra.datePicker.dateFormat": "س‌س‌س‌س/ي‌ي/ش‌ش",
   "Terra.datePicker.today": "اليوم",
-  "Terra.datePicker.openCalendar": "فتح التقويم"
+  "Terra.datePicker.openCalendar": "فتح التقويم",
+  "Terra.datePicker.dateFormatOrder": "2"
```

Comment:
What do "0", "1" and "2" signify? Can they be more meaningful - maybe throw in the actual formats ('MM-DD-YYYY', 'YYYY-MM-DD' and 'DD-MM-YYYY' respectively) in here?

Reply:
This is probably easier, but I'd be interested to get @benbcai 's input on it.